generate.sh

```diff
@@ -1,11 +1,12 @@
 #!/bin/bash
 
-while getopts g:t:m:u:w: option
+while getopts g:t:d:m:u:w: option
 do
  case "${option}"
  in
  g) GITURL=${OPTARG};;
  t) export DOCTHEME=${OPTARG};;
+ d) export DEBUG=${OPTARG};;
```

Comment:
valid concern. do you know of any other way for the child process to access parent's variables? also regarding concurrent execution, multiple calls to generate.sh would occur in different shells right? If that is the case, `export` shouldn't be a problem. AFAIK `export` doesn't modifies the parent's environment.